### ISSUE TITLE
Does not appear to work with React 16

### ISSUE THREAD

**Issue by kylegoetz** [ivos/react-forms-ui#2]
Including a TextArea and no other form element yields the error "Element ref was specified as a string (messages) but no owner was set. . . . "

Looking at the code for messages, ref is set to a String type, but [here](https://reactjs.org/docs/refs-and-the-dom.html#legacy-api-string-refs) we see this was deprecated a while back and I'm guessing now is no longer supported.

I do not know how to work with refs, so I can't make these fixes myself. But it does seem the lib is incompatible with React 16.

**Comment by ivos** [comment]
Yep, was written against React 15. As I no longer use it myself, I do not plan to upgrade. A merge request is however welcome.
